_data/tools.yml

```
@@ -1885,3 +1885,17 @@
   v2: true
   v3: true
   v3_1: true
+   
+- name: BlocklyAutomation
+  category: 
+    - code-generators
+    - sdk
+    - documentation
+    - testing
+  link: https://ignatandrei.github.io/BlocklyAutomation/
+  github: http://github.com/ignatandrei/BlocklyAutomation
+  language: Javascript / .NET
```

```suggestion
  language: SaaS
```

Would SaaS fit this better as it's not something you run locally right?

---

You can run this locally. There is a Microsoft Store app ( https://www.microsoft.com/en-us/p/blocklyautomation/9mxnwjt6zpd0 , will be updated monthly ) and a desktop application for Windows , updated more frequently - http://ba.serviciipeweb.ro/ . The desktop app has no CORS problems. So, no, it is not just  SAAS application